lib/vdsm/common/udevadm.py

```diff
@@ -53,7 +53,12 @@ def settle(timeout, exit_if_exists=None):
         logging.error("%s", e)
 
 
-def trigger(attr_matches=(), property_matches=(), subsystem_matches=()):
+def trigger(
+    path=None,
```

Comment:
Any reason to add the path first? Looks like all the callers are using key=value
so the order does not matter, but the command always add path at the end, so adding the parameter as the last parameter makes the python call mode like the actual command.

Reply:
No particular reason, I'll move it to the end